### DISCUSSION TITLE
Can not connect to web interface

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#8953] [Unsupported Versions] [answered]
Hello, I have a distributed setup with one manager and one forward node. At the moment the forward node is not setup but will be soon. The manager is setup and running, I have done so-allow and chose analyst and put in my ip address to whitelist so I may connect to the web interface. But I have not been able to reach the login page for the web interface. All I get is this message "This site can’t be reached Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_CONNECTION_TIMED_OUT

I went and checked the nginx logs and the error log is full of connection refused. I'm adding in the error.log and have redacted all important information from the file. 
[error.log](https://github.com/Security-Onion-Solutions/securityonion/files/9824172/error.log)

Any help will be grateful and also I am using FortiClient VPN I don't think that is the problem but I added it incase it could possibly be.

**Comment by Sahale** [comment]
Hey!

`connect() failed (111: Connection refused) while connecting to upstream`
It usually means that backend is not working. What does `# so-status` show on manager node?

**Reply by its-a-me-mario-the-og** [reply]
![image](https://user-images.githubusercontent.com/91745779/196958299-a62ef314-bb8c-4977-9f19-6676ed357b00.png)

This is what comes up when I do so-status, everything seems to be up and running.

**Comment by its-a-me-mario-the-og** [comment, marked as answer]
Nevermind we found the answer I forgot we are using NAT addresses and didn't whitelist the correct ip. Thank you for your help Sahale!

**Reply by DrGratz** [reply]
how did you whitelist the right IP address?

**Reply by OlexTratisky** [reply]
You can use so-firewall or so-allow or manually edit the yaml files (advanced mode)
